tests/integration/ssh/test_state.py

```diff
@@ -159,6 +160,22 @@ def test_state_run_request(self):
         check_file = self.run_function('file.file_exists', [SSH_SLS_FILE], wipe=False)
         self.assertTrue(check_file)
 
+    def test_state_running(self):
+        '''
+        test state.running with salt-ssh
+        '''
+        start_sls = self.run_function('state.sls', ['running', '&'],
+                                      wipe=False)
+        time.sleep(8)
+        get_sls = self.run_function('state.running', wipe=False)
+        ret = 'The function "state.pkg" is running as'
+        self.assertIn(ret, ' '.join(get_sls))
+
+        # make sure we wait until the earlier state is complete
+        while True:
+            if ret not in ' '.join(self.run_function('state.running', wipe=False)):
+                break
```

Comment:
Can we do a little more here to ensure that if we encounter a bug here that we won't just hang the test suite in this loop? Maybe change this so that the test fails if it stalls or something.